caliban/docker/build.py

```diff
@@ -454,6 +493,30 @@ def _extra_dir_entries(workdir: str, user_id: int, user_group: int,
   return ret
 
 
+def _resource_entries(
+    workdir: str,
+    uid: int,
+    gid: int,
+    resource_files: Optional[List[str]] = [],
```

Comment:
use `None` vs an actual list, in case anyone later modifies this to return `resource_files`